src/mds/agg_mds/adapters.py

```diff
@@ -881,7 +881,7 @@ class Gen3Adapter(RemoteMetadataAdapter):
     @retry(
         stop=stop_after_attempt(10),
         retry=retry_if_exception_type(httpx.TimeoutException),
-        wait=wait_random_exponential(multiplier=1, max=60),
+        wait=wait_random_exponential(multiplier=1, max=10),
```

Comment:
Why reduce this?

Reply:
Ahh! I wanted to set it back to its previous value since increasing the timeout between calls is not the solution to it. 

I assumed it was 10 seconds like all the other cases where this method was used. But I just found the previous value was 20. 

Will update this time to 20 seconds. 

Thanks for catching this. :)